hacspec/src/lib.rs

```diff
@@ -601,17 +608,112 @@ pub fn construct_state(
     )
 }
 
+fn parse_cipher_suites(
```

Comment:
Since the function is specific to parsing `SUITES_I` in `message_1` consider renaming it accordingly. e.g. `parse_suites_i`